calculator.js

```diff
@@ -15,7 +15,12 @@ const divide = ((num1, num2) => {
     return num1 / num2;
 });
 
-// OPERATE FUNCION WITH SWITCH STATEMENT
+// OPERATE FUNCTION WITH SWITCH STATEMENT
+/* 
+Wow! Eric tells me this is a great way to write this function. 
+It's so short and simple and nice. 
+Way to go, me.
```

Comment:
```suggestion
It is so short. It is so simple and nice. 
Way to go, me - Ellen Jaquette!
```

Reply:
I went a new route because you had a terrible suggestion. Terrible.